### PR TITLE
Initial iteration detail page

### PR DESCRIPTION
**Pull request by chicohernando**
Includes changes to some other pages like navigation, browser titles, and linking to pages within the app

### CODE REVIEW

**Comment by chicohernando** [comment]
This will be the last PR without pivotal tracker ticket numbers

File: src/src/Controller/IterationController.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+class IterationController extends AbstractController
+{
+    /**
+     * @Route("/project/{project_id<\d+>}/iteration/{iteration_id<\d+>}", name="iteration_detail")
+     */
+    public function detail(int $project_id, int $iteration_id)
+    {
+        $pivotal_tracker_client = new Client($_ENV['PIVOTAL_TRACKER_API_KEY'], $project_id);
+        $project = $pivotal_tracker_client->getProject($project_id, ['fields' => 'name']);
+        $iteration = $pivotal_tracker_client->getIteration($iteration_id, ['fields' => 'number,team_strength,accepted_points,effective_points,velocity,start,finish,stories(id,name,story_type,estimate,current_state,url,owner_ids),points,accepted,created,analytics']);
```
Comment (totallyquiche) [inline src/src/Controller/IterationController.php]: Just to make `fields` a little less cumbersome:

```suggestion
        $iteration_fields = [
            'number',
            'team_strength',
            'accepted_points',
            'effective_points',
            'velocity',
            'start',
            'finish',
            'stories(id,name,story_type,estimate,current_state,url,owner_ids)',
            'points',
            'accepted',
            'created',
            'analytics'
        ];
        
        $iteration = $pivotal_tracker_client->getIteration($iteration_id, [
            'fields' => implode(',', $iteration_fields)
        ]);
```

File: src/templates/base.html.twig [hunk 2]
```diff
@@ -339,12 +353,21 @@
...
                             </div>
+                            
                             <div id="projects" class="tab-pane {% if app.request.attributes.get( '_route' ) starts with 'project_' %}in active{% endif %} notika-tab-menu-bg animated flipInX">
                                 <ul class="notika-main-menu-dropdown">
                                     {{ render(controller('App\\Controller\\NavigationController::projectList')) }}
                                 </ul>
                             </div>
 
+                            {% if project is defined %}
+                                <div id="iterations" class="tab-pane {% if app.request.attributes.get( '_route' ) starts with 'iteration_' %}in active{% endif %} notika-tab-menu-bg animated flipInX">
```
Comment (totallyquiche) [inline src/templates/base.html.twig]: Should `in active` be `inactive`?

File: src/templates/iteration/detail.html.twig [hunk 3]
```diff
@@ -0,0 +1,151 @@
+{% extends 'base.html.twig' %}
+{% block title %}
+    {{ project.name }}: {{ iteration.start|date("m/d/Y") }} - {{ iteration.finish|date("m/d/Y") }} Iteration Details
+{% endblock %}
+{% block body %}
+    <div class="container">
+        <div class="row the-space-beneath">
```
Comment (totallyquiche) [inline src/templates/iteration/detail.html.twig:7]: What does `the-space-beneath` mean? :thinking:

**Review by totallyquiche — CHANGES_REQUESTED** [review]

File: src/templates/base.html.twig (hunk 2 above)
Comment (chicohernando) [reply src/templates/base.html.twig]: Looks like `in` doesn't belong at all.  I was just following what the template was doing.  I removed `in` from the two menu items above and tested and everything still works as I would expect.

File: src/templates/iteration/detail.html.twig (hunk 3 above)
Comment (chicohernando) [reply src/templates/iteration/detail.html.twig:7]: 😂 it is referring to the css 
```css
.the-space-beneath {
    margin-bottom: 20px;
}
```

I couldn't think of a better way, at the time, to explain that I wanted space beneath this HTML element.  I also didn't want to apply the margin to all `row`s.

Without the class:

Screen Shot 2020-03-02 at 8 38 03 PM

----

With the class:
Screen Shot 2020-03-02 at 8 38 16 PM

**Review by totallyquiche — APPROVED** [review]
LGTM

**Pull request merged**